dashboard/src/components/AppView/ChartUpdateInfo.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  checkCircleIcon,
+  circleArrowIcon,
+  ClarityIcons,
+  exclamationTriangleIcon,
+} from "@clr/core/icon-shapes";
+import * as React from "react";
+import { CdsIcon } from "../Clarity/clarity";
+
+import { IRelease } from "shared/types";
+
+ClarityIcons.addIcons(exclamationTriangleIcon, checkCircleIcon, circleArrowIcon);
+
+interface IChartInfoProps {
+  app: IRelease;
+}
+
+export default function ChartUpdateInfo(props: IChartInfoProps) {
+  const { app } = props;
+  // If update is not set yet we cannot know if there is
+  // an update available or not
+  if (app.updateInfo) {
```

Comment:
Could be a little easier to read calling out to a separate function: `const updateInfo = getUpdateInfoForApp(app)` to be returned below? Fine either way, but either way, there is quite a lot of logic here that should be tested.

Reply:
_now it should be more readable, and covered by tests_